enclave-tls/src/attesters/nullattester/cleanup.c

```diff
@@ -7,7 +7,7 @@
 #include <enclave-tls/log.h>
 #include <enclave-tls/attester.h>
 
-enclave_attester_err_t nullattester_cleanup(enclave_attester_ctx_t *ctx)
+enclave_attester_err_t nullattester_cleanup(__attribute__((unused)) enclave_attester_ctx_t *ctx)
```

Comment:
Why sgx build can accept `__attribute__((unused))` here unlike libattester_null_init()?

Reply:
libattester_null_init doesn't use __attribute__((unused)). Under host mode, it use __attribute__((constructor))

Reply:
I mean in sgx build libattester_null_init seems to recognize attribute, so you use the conditional compiling. So why nullattester_cleanup in sgx build can use attribute.

Reply:
yes, I mean it should be attribute((unused)) can be used in sgx build(this attribute((unused)) is used in many functions under sgx build), but "constructor" should not.